api/V1/recordingUtil.ts

```diff
@@ -1098,6 +1106,37 @@ function addAudioBaitRow(out: any, audioBait: Event) {
     ""
   ]);
 }
+
+async function sendAlerts(recID: number) {
+  const recording = (await models.Recording.getForAdmin(recID)) as any;
```

Comment:
As discussed, not sure that having the `getForAdmin` abstraction is worthwhile until we have more consumers of that functionality - probably better to just inline the logic here.